src/main/kotlin/no/nav/syfo/behandler/api/BehandlerApi.kt

```diff
@@ -56,6 +56,17 @@ fun Route.registerBehandlerApi(
                 call.respond(behandlere.toBehandlerDTOListUtenRelasjonstype())
             }
         }
+        post(search) {
```

Comment:
POSt er kanskje mest riktig for søk uansett? Evt kunne dette vært en GET med query-params, men da må navn sendes rundt i klartekst og det er kanskje ikke så bra det heller.

Reply:
Ja, det kan være vi burde gå over til å bruke denne vi også. Virker som det Kelvin har problemer med er encoding av headers.